debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -710,6 +710,16 @@ public static EventProcessingFailureHandlingMode parse(String value) {
                                                                       + "of the snapshot; in such cases set this property to 'false'.")
                                                               .withDefault(true);
 
+    public static final Field SNAPSHOT_PARALLEL = Field.create("snapshot.parallel")
+                                                       .withDisplayName("Snapshot newly added tables in parallel to normal binlog reading")
+                                                       .withType(Type.BOOLEAN)
```

Comment:
Do you think we might ever want other kinds of "snapshot" processing? If so, might we want this to be

    snapshot.new.tables=parallel

where the values are `parallel` and `off` (or `disabled`)?

Reply:
Ah, I like that. I will change it.